tests/openwisp2/sample_firmware_upgrader/migrations/0008_nullable_upgrade_operation_image.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.5 on 2021-01-21 16:57
```

Comment:
@nemesisdesign I saw the number of migrations in sample app and many are just tweaking some attributes. Do you think we should squash such migration files into one or just update the initial migrations for sanity(just for sample app)?

Reply:
yes in general I ask to re-generate the migrations but in this case since the migrations are many we could do it in a separate issue:
https://github.com/openwisp/openwisp-firmware-upgrader/issues/119